rfc6962-bis.xml

```diff
@@ -541,9 +541,7 @@ ranges.
         v2(1), (255)
     } Version;
 
-    struct {
-        opaque key_id[HASH_SIZE];
-    } LogID;
+    opaque LogID&lt;1..2^8-1&gt;;
```

Comment:
suggest increasing to two bytes as there are already ~10 logs, we'd like to be able to run test logs as well.